Makefile

```diff
@@ -211,6 +211,7 @@ build: $(CLI_BIN) ## Build
 tests: $(CLI_BIN) ## Run tests
 	@echo Running tests... >&2
 	@go test ./... -race -coverprofile=coverage.out -covermode=atomic
+	@go tool cover -html=coverage.out
```

Comment:
I think you will need to output to a file

Reply:
yeah if i don't output to a file it open up in the browser.

Reply:
yup, looks like ci is green though 🤷

Reply:
yes. but Now I have pushed the change LOL

Reply:
not pushed this can be merged